core/orch_test.go

```diff
@@ -290,7 +290,7 @@ func TestSelectTranscoder(t *testing.T) {
 
 	// register transcoders, which adds transcoder to liveTranscoders and remoteTranscoders
 	wg := newWg(1)
-	go func() { m.Manage(strm, 2) }()
+	go func() { m.Manage(strm, 1) }()
```

Comment:
The second argument is the transcoder's capacity. Any particular reason for this change, if I'm not mistaken it doesn't affect this test.

Reply:
We are asserting that no transcoders are returned if all are at capacity [here](https://github.com/livepeer/go-livepeer/pull/1849/files/b248bef1940018c77b7b4e4d0a317865484a6c0c#diff-9734c79b7138aa52372c8c9714be7aba54b219fb23099883e1533f44d9f79385R331), which is why it was changed to capacity 1. 
But I just realized that [this test](https://github.com/livepeer/go-livepeer/pull/1849/files/b248bef1940018c77b7b4e4d0a317865484a6c0c#diff-9734c79b7138aa52372c8c9714be7aba54b219fb23099883e1533f44d9f79385R331) can also pass because the previous transcoder is at full load, and not particularly because it is a new session. Maybe I should change the capacity of the transcoder initialized after this one and then add another selectTranscoder with a new session before asserting for "no transcoders available". What do you think?

Reply:
Oh I see, it mimics the behaviour of the old test but with the updated behaviour of the actual code, gotcha. That's fine as is, however if the tests checks for a _new_ transcoder we could simply delete the current session before we do that part of the test.

Reply:
Yup. Just pushed a change to the previous commit, I deleted the current session before that test, however I had to add the load again before asserting for "no transcoders available". Let me know if you recommend any other changes.